### PR TITLE
IS-1167 Support Indy 1.8.1 / IS-1175  Use new logging pattern

### PR DESCRIPTION
**Pull request by Artemkaaas**
@cam-parra @KitHat @jovfer @mattraffel , please review

### CODE REVIEW

File: libsovtoken/src/logic/api_internals/add_request_fees.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //! TODO ???
```
Comment (mattraffel) [inline libsovtoken/src/logic/api_internals/add_request_fees.rs]: I would like to see this source file commented to keep consistent with other files and maintain the generated documentation

File: libsovtoken/src/logic/indy_sdk_api/crypto.rs [hunk 2]
```diff
@@ -0,0 +1,107 @@
+use {ErrorCode, IndyHandle};
```
Comment (mattraffel) [inline libsovtoken/src/logic/indy_sdk_api/crypto.rs]: I would like to see this file commented, same as previous comment

File: libsovtoken/src/utils/callbacks.rs [hunk 3]
```diff
@@ -0,0 +1,133 @@
+//!
+
```
Comment (mattraffel) [inline libsovtoken/src/utils/callbacks.rs]: is this file even used any more?

File: libsovtoken/src/logic/api_internals/add_request_fees.rs (hunk 1 above)
Comment (Artemkaaas) [reply libsovtoken/src/logic/api_internals/add_request_fees.rs]: I'm afraid I am not the best person to describe sovtoken files.

File: libsovtoken/src/utils/callbacks.rs (hunk 3 above)
Comment (Artemkaaas) [reply libsovtoken/src/utils/callbacks.rs]: yes. these functions use in `logic/indy_sdk_api` a lot to call indy-sys functions

**Pull request merged**